src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball() {
+
+    }
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String toString(Ball ball) {
```

Comment:
toString is a method which you have to override from Object class. It do not accepts any parameters. It returns internal presentation of our current object.

https://www.javatpoint.com/understanding-toString()-method

Reply:
:0